include/NovelRT/Maths/GeoVector.h

```diff
@@ -99,9 +107,21 @@ namespace NovelRT::Maths {
       return GeoVector<T>(getVec2Value() / other);
     }
 
-    void rotateToAngleAroundPoint(T angleRotationValue, const GeoVector<T>& point) {
+    void rotateToAngleAroundPoint(T angleRotationValue, const GeoVector<T>& point) noexcept {
       setVec2Value(glm::rotate((getVec2Value() - point.getVec2Value()), glm::radians(angleRotationValue)) + point.getVec2Value());
     }
+
+    bool epsilonEquals(const GeoVector<T>& other, const GeoVector<T>& epsilonValue) const noexcept {
+      return glm::all(glm::equal(getVec2Value(), other.getVec2Value(), epsilonValue.getVec2Value()));
+    }
+
+    static GeoVector<T> zero() {
```

Comment:
why not have these as const?

Reply:
statics can't have those sorts of qualifiers, according to the compiler.

Reply:
no, I meant as a const field rather than a method

Reply:
As per the discussion in discord, this was causing compiler errors. Will be resolved at a later date.